ansible/roles/debops.grafana/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Install requested packages
+  package:
+    name: '{{ q("flattened", grafana__base_packages
+                             + grafana__packages) }}'
+    state: 'present'
+  register: grafana__register_packages
+  until: grafana__register_packages is succeeded
+  notify: [ 'Restart grafana' ]
+
+- name: Divert the original grafana configuration file
+  command: dpkg-divert --quiet --local --divert /etc/grafana/grafana.ini.dpkg-divert --rename /etc/grafana/grafana.ini
+  args:
+    creates: '/etc/grafana/grafana.ini.dpkg-divert'
+
+- name: Configure main grafana config file
+  template:
+    src: 'grafana.ini.j2'
+    dest: '/etc/grafana/grafana.ini'
+    owner: 'root'
+    group: 'adm'
+    mode: '0644'
```

Comment:
Kind of weird group selection here... Is the configuration file really owned by `root:adm`? Also, are there any passwords in that configuration file? In this case you should use stricter mode.

Reply:
changed group to grafana and permissions to 0640